src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,39 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.models.Manufacturer;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
 
+    public static void main(String[] args) {
+        ManufacturerDao manufacturerDao = (ManufacturerDao) injector
+                .getInstance(ManufacturerDao.class);
+        //create method
+        Manufacturer manufacturer1 = new Manufacturer();
+        manufacturer1.setName("BMW");
+        manufacturer1.setCountry("Poland");
+        System.out.println(manufacturerDao.create(manufacturer1));
+        // delete method
+        System.out.println("Delete");
+        System.out.println(manufacturerDao.delete(4L));
```

Comment:
```suggestion
        System.out.println("Delete");
        System.out.println(manufacturerDao.delete(bmwManufacturer.getId()));
```

let's get rid of the magic numbers in the solution to make it more unique